jni/rubicon.c

```diff
@@ -1053,22 +1056,16 @@ JNIEXPORT void JNICALL Java_org_beeware_rubicon_Python_stop(JNIEnv *env, jobject
  * case, it returns a boxed java.lang.Integer.
  *************************************************************************/
 JNIEXPORT jobject JNICALL Java_org_beeware_rubicon_PythonInstance_invoke(JNIEnv *env, jobject thisObj, jobject proxy, jobject method, jobjectArray jargs) {
-    LOG_D("Invocation");
```

Comment:
Removing these log lines is definitely worthwhile for cleanup; my concern is that if something breaks, we've lost the ability to diagnose where the problem lies. Is there any value in retaining *some* of these logging lines - even if they're demoted to V level - so that in an emergency, we can see where the connection breaks? 

I suppose the meta-question here: What's the cost of a logging operation?

Reply:
I retained the one `Native invocation: ...` log line because I agree it's nice to easily know where something has gone wrong.

I don't really know the cost of logging :(. I can try to investigate it. I'm OK re-adding this with `V` level if you like, but I personally think the remaining `LOG_D` call in the function makes it unnecessary. Curious for your thoughts on that.

Reply:
Specifically I think this https://github.com/beeware/rubicon-java/pull/44/files/fcf62e06eba028a6873809e96be5601a1af03478#diff-dad8865bf1af7a7aea8109818787586bR1069 is the line that makes it fine to remove the `LOG_D` lines.

Reply:
Yeah - I think you're right. The previous logging was overkill, and as long as we've got some indicator that functions are being invoked, we should be good.